Audiobook Player/PlayerManager.swift

```diff
@@ -402,11 +414,12 @@ extension PlayerManager: AVAudioPlayerDelegate {
             if Int(audioplayer.currentTime) >= chapter.start {
                 self.currentChapter = chapter
                 let chapterString = "Chapter \(chapter.index) of \(self.chapterArray.count)"
-                //notify
                 let userInfo = ["chapterString": chapterString, "fileURL": self.currentBooks.first!.fileURL] as [String: Any]
 
                 //notify
-                NotificationCenter.default.post(name: Notification.Name.AudiobookPlayer.updateChapter, object: nil, userInfo: userInfo)
+                DispatchQueue.main.async {
```

Comment:
`updateCurrentChapter` is called from inside `updateTimer` so it shouldn't have a problem either